tests/neg/i4058.scala

```diff
@@ -4,6 +4,6 @@ class C(abstract val a: Int) // error
 class D {
   def f(sealed a: Int) = 0 // error
   def g(lazy a: Int) = 0 // error
-  def g(ov erride a: Int) = 0 //error
+  def g(override a: Int) = 0 // error
```

Comment:
Hadn't found `testCompilation i4058` yet, my bad for not reading http://dotty.epfl.ch/docs/contributing/testing.html 😅